helm/hpcc/templates/_helpers.tpl

```diff
@@ -351,26 +388,51 @@ The plane will generate a volume if it matches either an includeLabel or an incl
 {{- $previousMounts := dict -}}
 {{- range $plane := $planes -}}
  {{- if not $plane.disabled -}}
-  {{- if or ($plane.pvc) (hasKey $plane "storageClass") -}}
-   {{- if not (hasKey $previousMounts $plane.prefix) -}}
-    {{- $mountpath := $plane.prefix -}}
-    {{- if or (has $plane.category $includeCategories) (has $plane.name $includeNames) }}
-     {{- $pvc := hasKey $plane "pvc" | ternary $plane.pvc (printf "%s-%s-pvc" (include "hpcc.fullname" $) $plane.name) -}}
-     {{- $num := int ( $plane.numDevices | default 1 ) -}}
-     {{- if le $num 1 }}
+  {{- if or (has $plane.category $includeCategories) (has $plane.name $includeNames) }}
+
+   {{- if or ($plane.pvc) (hasKey $plane "storageClass") -}}
+    {{- if not (hasKey $previousMounts $plane.prefix) }}
+     {{- $pvc := hasKey $plane "pvc" | ternary $plane.pvc (printf "%s-%s-pvc" (include "hpcc.fullname" $) $plane.name) }}
+     {{- $numMounts := int ( $plane.numMounts | default $plane.numDevices | default 1 ) }}
+     {{- if le $numMounts 1 }}
 - name: {{ lower $plane.name }}-pv
   persistentVolumeClaim:
     claimName: {{ $pvc }}
      {{- else }}
-      {{- range $elem := until $num }}
+      {{- range $elem := until $numMounts }}
 - name: {{ lower $plane.name }}-pv-many-{{- add $elem 1 }}
   persistentVolumeClaim:
     claimName: {{ $pvc }}-{{- add $elem 1 }}
       {{- end }}
-     {{- end -}}
+     {{- end }}
+     {{- $_ := set $previousMounts $plane.prefix true }}
+    {{- end }}
+   {{- end }}
+
+   {{- /*Generate entries for each alias of the plane*/ -}}
+   {{- $curAlias := dict "num" 1 -}}
+   {{- range $alias := $plane.aliases | default list }}
+    {{- if $alias.pvc -}}
+     {{- if not (hasKey $previousMounts $alias.prefix) }}
+      {{- $pvc := $alias.pvc }}
+      {{- $numMounts := int ( $alias.numMounts | default $plane.numDevices | default 1 ) }}
+      {{- if le $numMounts 1 }}
+- name: {{ lower $plane.name }}-pv-alias-{{ $curAlias.num }}
+  persistentVolumeClaim:
+    claimName: {{ $pvc }}
+      {{- else }}
+       {{- range $elem := until $numMounts }}
+- name: {{ lower $plane.name }}-pv-alias-{{ $curAlias.num }}-many-{{- add $elem 1 }}
```

Comment:
trivial: {{- add $elem 1 }} , I don't think the remove leading white space syntax needed here, harmless, but made me look.